pkg/services/ngalert/instance.go

```diff
@@ -0,0 +1,84 @@
+package ngalert
+
+import (
+	"fmt"
+	"time"
+)
+
+// AlertInstance represents a single alert instance.
+type AlertInstance struct {
+	AlertDefinitionOrgID int64  `xorm:"org_id"`
+	AlertDefinitionUID   string `xorm:"alert_definition_uid"`
```

Comment:
Maybe Since these are a property of the go `AlertInstance` and the `alert_instance` table. We can Drop the `Alert` prefix from these properties in both Go and the db. So just `DefinitionOrgID` and `DefinitionUID`